sdk/storage/Azure.Storage.DataMovement.Blobs/README.md

```diff
@@ -207,29 +247,25 @@ Upload a directory as a specific blob type.
 ```C# Snippet:SimpleDirectoryUpload
 DataTransfer dataTransfer = await transferManager.StartTransferAsync(
     sourceResource: files.FromDirectory(sourcePath),
-    destinationResource: blobs.FromClient(
-        blobContainerClient,
+    destinationResource: blobs.FromContainer(
+        blobContainerUri,
         new BlobStorageResourceContainerOptions()
         {
             // Block blobs are the default if not specified
             BlobType = BlobType.Block,
             BlobDirectoryPrefix = optionalDestinationPrefix,
-        }),
-    transferOptions: options);
-await dataTransfer.WaitForCompletionAsync();
+        }));
 ```
 
 ### Download
 
 A download takes place between a blob `StorageResource` as source and local file `StorageResource` as destination.
 
-Download a block blob.
+Download a blob.
```

Comment:
Was wondering if there's a way to also make this a header, but not bigger than the ### header. Just to make it stand out somehow that this is a title of the sample they're about to read.